nixos/modules/installer/cd-dvd/installation-cd-graphical-base.nix

```diff
@@ -8,6 +8,30 @@ with lib;
 {
   imports = [ ./installation-cd-base.nix ];
 
+  users.extraUsers.live = {
+    isNormalUser = true;
+    uid = 1000;
```

Comment:
We probably shouldn't use UIDs in the `>1000` range since that's reserved for users. Maybe just leave out the UID?